moderation/schema.go

```diff
@@ -51,6 +51,8 @@ CREATE TABLE IF NOT EXISTS moderation_configs (
 	warn_include_channel_logs BOOLEAN,
 	warn_send_to_modlog BOOLEAN,
 	warn_message TEXT,
+	unwarn_send_to_modlog BOOLEAN,
```

Comment:
From what I understand, you will instead need to add these as alterations to the table after defining the table
in the schema. If you add them in here, on existing hosts, it will simply note that the `moderation_configs` table already exists and move on. If you instead insert before line 81 
```txt
`ALTER TABLE moderation_configs ADD COLUMN IF NOT EXISTS unwarn_send_to_modlog BOOLEAN NOT NULL DEFAULT true;`
```
then it will be able to set the new column into the existing table while also assigning a default value to every existing row (every 
guild’s existing configuration). This is what I was referring to [here](https://discord.com/channels/166207328570441728/578976767058903057/1316992372084834449).